content/100-getting-started/index.mdx

```diff
@@ -1,108 +1,163 @@
-
-<ButtonLink
-  color="dark"
-  type="primary"
-  href="./getting-started/setup-prisma/add-to-existing-project/relational-databases-typescript-postgresql"
->
-  Existing SQL project
-</ButtonLink>
-
-<ButtonLink
-  color="dark"
-  type="primary"
-  href="./getting-started/setup-prisma/add-to-existing-project/mongodb-typescript-mongodb"
->
-  Existing MongoDB project
-</ButtonLink>
-
-To learn how to **create a new project** with Prisma from scratch, choose between SQL and MongoDB. Both tutorials will guide you through creating a project, connecting your database, and querying your data.
-
-<ButtonLink
-  color="dark"
-  type="primary"
-  href="./getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgresql"
->
-  New SQL project
-</ButtonLink>
-
-<ButtonLink
-  color="dark"
-  type="primary"
-  href="./getting-started/setup-prisma/start-from-scratch/mongodb-typescript-mongodb"
->
-  New MongoDB project
-</ButtonLink>
+[**<Bolt />Prisma Accelerate**](/accelerate/what-is-accelerate) is a global database cache with scalable connection pooling.
 
-If you're interested in learning how Prisma integrates with your favorite library or framework, such as Next.js, Express, Apollo GraphQL, NestJS, etc, you can check out our **ready-to-run examples**.
+[**<SignalStream />Prisma Pulse**](/pulse/what-is-pulse) allows you to build reactive, real-time applications in a type-safe manner.
 
-<ButtonLink
-  color="dark"
-  type="primary"
-  href="https://github.com/prisma/prisma-examples/"
->
-  Examples
-</ButtonLink>
+</BorderBox>
 
-### Get started with Prisma Accelerate
-
-Add **Accelerate** to your app to use connection pooling and global database caching.
-
-<ButtonLink color="dark" type="primary" href="/accelerate/getting-started">
-  Get started with Accelerate
-</ButtonLink>
-
-### Get started with Prisma Pulse
-
-Add **Pulse** to your app to subscribe to real-time updates from your database using Prisma Client.
-
-<ButtonLink color="dark" type="primary" href="/pulse/getting-started">
-  Get started with Pulse
-</ButtonLink>
-
-## Prisma with different tooling
-
-Prisma can be used with a wide range of tooling and frameworks, the following links outline how to get started with some of them:
-
-- [Prisma with Next.js](https://www.prisma.io/nextjs)
-- [Prisma with GraphQL](https://www.prisma.io/graphql)
-- [Prisma with Apollo](https://www.prisma.io/apollo)
-- [Prisma with Nestjs](https://www.prisma.io/nestjs)
-- [Prisma with Express](https://www.prisma.io/express)
-- [Prisma with hapi](https://www.prisma.io/hapi)
-
-<Admonition type="info">
-
-Join our thriving community on [Slack](https://slack.prisma.io/), [Discord](https://discord.gg/KQyTW2H5ca), and [GitHub](https://github.com/prisma/prisma) for help and ideas.
+</TopBlock>
 
-</Admonition>
+## <Database color="#5A67D8" width="21" height="24" /> Prisma ORM
+
```

Comment:
It may be redundant but visually and also to add more context, I personally would probably prefer if we had another short blurb after each "product heading", e.g. here for Prisma ORM:"

```suggestion

Add Prisma ORM to your application in a few minutes to start modeling your data, run schema migrations and query your database.

```

If folks generally agree, I can write up a short sentence both for Accelerate and Pulse as well.

Reply:
I added three sentences to the sections:

- Add Prisma ORM to your application in a few minutes to start modeling your data, run schema migrations and query your database.
- Make your app faster by scaling your database connections and caching database results at the edge with Prisma Accelerate.
- Build real-time applications by subscribing to data changes in your database using Prisma Pulse.

Reply:
I like it. Do you think it overlaps too much with the text in the Get Started boxes?

Reply:
There's certainly a bit of redundancy but I don't think it hurts:
- the text at the top is about _what_ the product is; these other sentences are written as _CTAs_ that rephrase the things from the top and point to the value of the tools in slightly different ways. 
- I actually think this kind of redundancy can be helpful because 
  - not everyone may read the sentences at the top, so repeating information again in the context of each product won't hurt IMO
  - for some people the phrasing of the sentences on the top may resonate more, for other people the other sentences may resonate more